tests/test_fmi3.py

```diff
@@ -233,12 +249,63 @@ def test_get_default_experiment_stop_time(self):
         fmu = load_fmu(FMI3_REF_FMU_PATH / "VanDerPol.fmu")
         assert fmu.get_default_experiment_stop_time() == 20.0
 
-
     def test_get_default_experiment_tolerance(self):
         """Test retrieving default experiment tolerance. """
         fmu = load_fmu(FMI3_REF_FMU_PATH / "VanDerPol.fmu")
         assert fmu.get_default_experiment_tolerance() == 0.0001
 
+    def test_get_states_list(self):
+        """Test retrieving states list and check its attributes. """
+        fmu = load_fmu(FMI3_REF_FMU_PATH / "VanDerPol.fmu")
+        states = fmu.get_states_list()
+
+        assert len(states) == 2
+        x0 = states['x0']
+        x1 = states['x1']
+
```

Comment:
Could add another typecheck for x0, x1 here.

Reply:
Good catch! Fixed in latest commit.